packages/client/src/test.cairo

```diff
@@ -22,13 +23,15 @@ fn test(mut arguments: Span<felt252>) {
     let Args { mut chain_state, blocks, expected_chain_state } = Serde::deserialize(ref arguments)
         .expect('Failed to deserialize');
 
+    let mut state: State = State { chain_state: chain_state, utreexo_state: Default::default(), };
```

Comment:
temporary solution while python scripts are not updated.
Will only work if we test one isolated block, or a batch of block starting from genesis block as instantiate a default `utreexo_state` in the `state`